src/redux/auth-reducer.js

```diff
@@ -0,0 +1,75 @@
+import {authAPI} from "../api/api";
+import {stopSubmit} from "redux-form";
+
+const SET_USER_DATA = 'SET_USER_DATA';
+
+let initialState = {
+    accessToken: "",
+    accessTokenExpiresIn: "",
+    refreshToken: "",
+    refreshTokenExpiresIn: "",
+    login: "",
+    email: "",
+    password: "",
+    isAuth: false
+};
+
+const authReducer = (state = initialState, action) => {
+
+    switch (action.type) {
+
+        case SET_USER_DATA:
+            return {
+                ...state,
+                ...action.payload
```

Comment:
You put here `...action.payload` but in the action actually 
```
{
    type: SET_USER_DATA,
    data: {accessToken, accessTokenExpiresIn, refreshToken, refreshTokenExpiresIn, isAuth}
}
```
data...payload...payload...data...

Reply:
fix